src/test/java/org/openhab/binding/jrule/internal/rules/timers/TargetItem.java

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Could be mocked or it is used multiple times?

Reply:
Could probably be mocked, just a bit messy to mock the static JRuleItemRegistry